DfciPkg/Docs/QemuQ35Pkg/ReadMe.md

```diff
@@ -0,0 +1,239 @@
+# Building and Running QemuQ35Pkg with DFCI
+
+QemuQ35Pkg virtual machine environment is available at
+[here](https://github.com/microsoft/mu_tiano_platforms.git).
+
+This document describes how to build and Run a QemuQ35Pkg virtual device to
+exercise the DFCI test cases.
+
+## Getting started
+
+You will need the following:
+
+1. Knowledge that the sub directory DfciDutFiles is used for both the build and run scripts for
+Q35, the emulated USB drives used to install windows, and setup the DFCI Test solution into the QemuQ35Pkg VM.
+2. An install image for Windows 11 (available when you purchase a license for Windows 11).

```

Comment:
I believe it is possible to download a windows 11 iso without a valid license. Is the license required to enable DFCI support?

Reply:
Just following the party line.  I don't think I should be the one publishing workarounds, even though there are some that are published - and the work arounds have to change as release of Windows change.  As for DFCI, there is no special license for use.